third_party/android_deps/libs/com_facebook_shimmer/README.chromium

```diff
@@ -0,0 +1,4 @@
+Name: Shimmer-android
+URL: https://github.com/facebook/shimmer-android
+License: BSD
```

Comment:
This will work, but slightly better would be to use a valid [SPDX identifier](https://spdx.org/licenses/) here: `BSD-3-Clause`.